lib/packer.js

```diff
@@ -88,68 +94,54 @@ export default function packer({ records, entrypoint, bytecode }) {
           }
         } else if (store === STORE_LINKS) {
           if (Array.isArray(value)) {
-            const buffer = Buffer.from(JSON.stringify(value));
+            const dedupedValue = [...new Set(value)];
+            log.debug('files & folders deduped = ', dedupedValue);
+            const buffer = Buffer.from(JSON.stringify(dedupedValue));
             stripes.push({ snap, store, buffer });
           } else {
             assert(false, 'packer: bad STORE_LINKS');
           }
         } else if (store === STORE_STAT) {
           if (typeof value === 'object') {
-            // reproducible
-            delete value.atime;
-            delete value.atimeMs;
-            delete value.mtime;
-            delete value.mtimeMs;
-            delete value.ctime;
-            delete value.ctimeMs;
-            delete value.birthtime;
-            delete value.birthtimeMs;
-            // non-date
-            delete value.blksize;
-            delete value.blocks;
-            delete value.dev;
-            delete value.gid;
-            delete value.ino;
-            delete value.nlink;
-            delete value.rdev;
-            delete value.uid;
```

Comment:
Why?

Reply:
Because of simplification in walker.js


 -    record[STORE_STAT] = await fs.stat(record.file);

Replace by; 
      const valueStat = await fs.stat(record.file);

      const value = {
        mode: valueStat.mode,
        size: valueStat.isFile() ? valueStat.size : 0,
        //
        isFileValue: valueStat.isFile(),
        isDirectoryValue: valueStat.isDirectory(),
        isSocketValue: valueStat.isSocket(),
        isSymbolicLinkValue: valueStat.isSymbolicLink(),
      };
      record[STORE_STAT] = value;
    }